benchmark/run.js

```diff
@@ -45,8 +47,7 @@ if (format === 'csv') {
   console.log('"filename", "configuration", "rate", "time"');
 }
 
-(function recursive(i) {
-  const filename = benchmarks[i];
+async function runBenchmark(filename) {
```

Comment:
Should you mark as async when you don't await inside it?

Reply:
Removed